app/config/default.json

```diff
@@ -19,6 +19,6 @@
     "maxAttempts": "3",
     "port": "3000",
     "smtpHost": "apps.smtp.gov.bc.ca",
-    "blockDoNotReplySender": false
+    "blockDoNotReplySender": true
```

Comment:
Do we want potentially instances of CHES that we don't manage to be blocking donotreply by default as well?